src/views/name.njk

```diff
@@ -51,6 +53,24 @@
       nameField.value = nameField.value.trim();
     });
   });
+
+  const content = document.getElementById('content');
+  const whichType = sessionStorage.getItem("which-type");
+  const whichTypeDb = content.dataset.type;
+
+  const value = (whichType && whichType !== whichTypeDb) ? whichType : whichTypeDb;
+
+  if (value == "registerLp"){
```

Comment:
not sure if you have merged latest from main but values have been changed to things like 
     value: "LP",
in main branch.

Reply:
I pulled main before doing that but I don't think it was merge yet
any way, it is a draft and when the decision is taken I need to resolve the conflicts